source/tool/chipsec/helper/linux/helper.py

```diff
@@ -99,7 +103,30 @@ def __del__(self):
 # Driver/service management functions
 ###############################################################################################
 
+    # This function load CHIPSEC driver. (implement functionality from run.sh)
+    def load_chipsec_module(self):
+        page_is_ram = ""
+        if self.GET_PAGE_IS_RAM:
+            page_is_ram = self.get_page_is_ram()
+            if not page_is_ram:
+                if logger().VERBOSE:
+                    logger().log("Cannot find symbol 'page_is_ram'")
+        driver_path = os.path.join(chipsec.file.get_main_dir(), ".." , "drivers" ,"linux", "chipsec.ko" )
+        subprocess.check_output( "insmod " + driver_path + " a1=0x%s"%page_is_ram     , shell=True)
+        subprocess.check_output( "chown root:root " + self.DEVICE_NAME            , shell=True)
```

Comment:
Is that necessary? What is the default kernel behaviour? Have you observed older config that behaves differently?

If it is, I suggest using `os.chown` and `os.chmod`.